apm-configuration/scala.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Scala"
+---
+
+# Prerequisites
+
+<Steps>
+    <Step title="Java APM">
+        The Java APM must be installed prior to installing Scala. Learn how to install the Java APM [here](/apm-configuration/java)
```

Comment:
IMO, `The Java APM must be installed prior to installing Scala.` this doesn't sounds proper.

Because technically, there is no installation process of Java APM. 
Scala just needed the latest version of `middleware-javaagent-{version}.jar` file.